Upgrade references to Wrangler v4+ for 'using' (#20797)
cloudflare/cloudflare-docs: src/content/docs/workers/runtime-apis/rpc/lifecycle.mdx

### How to use the `using` declaration in your Worker

> [Wrangler](/workers/wrangler/) v4+ supports the `using` keyword natively. If you are on an earlier version of Wrangler, you can alternatively call dispose of resources manually.

Reviewer: ```suggestion
[Wrangler](/workers/wrangler/) v4+ supports the `using` keyword natively. If you are using an earlier version of Wrangler, you will need to manually dispose of resources instead.
```

Reviewer: @GregBrimble Sorry I didn't realize you had auto-merge on. I think you have a typo here. Can you confirm my suggestion here and I'll make another PR to fix it?
